app/src/backend/utils.ts

```diff
@@ -133,7 +133,7 @@ function unixBootstrapCommand(): string {
     /* ignore */
   }
 
-  return paths.length != 0 ? `export PATH=${paths.join(':')}:$PATH;` : null;
+  return paths.length != 0 ? `export PATH=\"${paths.join(':')}:$PATH\";` : null;
```

Comment:
No need to escape double quotes in template literals.

Reply:
Right. Was experimenting with something and forgot to revert.